cashu/wallet/wallet.py

```diff
@@ -424,13 +429,29 @@ async def request_mint(self, amount: int, memo: Optional[str] = None) -> MintQuo
         Returns:
             MintQuote: Mint Quote
         """
-        mint_quote_response = await super().mint_quote(amount, self.unit, memo)
+        privkey, pubkey = await self.get_quote_ephemeral_keypair()

```

Comment:
Can we move `get_quote_ephemeral_keypair` also to `nut19.py`?

Reply:
So if we later want to add derivation from seed+counter functionality we might want to keep it inside the wallet class.

Reply:
In general depends on the state of the wallet so...